### PR TITLE
Raise an error if user target `SWIFT_VERSION` is missing

### PR DESCRIPTION
**Pull request by dnkoutso**
https://github.com/CocoaPods/CocoaPods/issues/7770

### CODE REVIEW

**Comment by CocoaPodsBarista** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please include a CHANGELOG entry to credit yourself! <br />
You can find it at <a href="https://github.com/CocoaPods/CocoaPods/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>

Here's an example of your CHANGELOG entry:

```markdown
* Raise an error if user target `SWIFT_VERSION` is missing  
  [dnkoutso](https://github.com/dnkoutso)
  [#issue_number](https://github.com/CocoaPods/CocoaPods/issues/issue_number)
```

*note*: There are two invisible spaces after the entry's text.


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by dnkoutso** [comment]
Argh validator issues.................!

**Comment by dnkoutso** [comment]
Should go green.

File: lib/cocoapods/validator.rb [hunk 1]
```diff
@@ -485,7 +486,6 @@ def add_app_project_import
       app_target = app_project.targets.first
       pod_target = @installer.pod_targets.find { |pt| pt.pod_name == spec.root.name }
       Pod::Generator::AppTargetHelper.add_app_project_import(app_project, app_target, pod_target, consumer.platform_name, use_frameworks)
-      Pod::Generator::AppTargetHelper.add_swift_version(app_target, swift_version)
```
Comment (dnkoutso) [inline lib/cocoapods/validator.rb]: this is actually better to happen _earlier_. We often validated targets and such and set the `SWIFT_VERSION` *afterwards*. All existing tests pass.

**Pull request merged**